tests/manual/test-procedure.rst

```diff
@@ -0,0 +1,282 @@
+Test Procedure tries to balance coverage vs test duration.
+
+
+Nomenclature and Definitions
+----------------------------
+
+- **Python Editor**: This constitutes the whole web app.
+
+- **Text Editor**: The text or code editor (using the Ace framework) inside the
+  Python Editor.
+
+- **Load by editor drag&drop**: Load a file into the Python Editor by dragging
+  a file from the user OS, to the Text Editor area.
+
+- **Load by drop area drag&drop**: Load a file into the Python Editor by
+  clicking on the "Load button" and then dragging a file from the user OS to
+  the large grey box that says "Drag and drop a .hex or .py file in here to
+  open it".
+
+- **Load by file picker**: Load a file into the Python Editor by clicking on
+  the "Load button", then clicking on the "Or pick a file" link, then clicking
+  on the "Browse" button, selecting a file in the browser file picker, and
+  finally clicking on the "Load" button.
+
+- **Load by any method**: Load a file into the Python Editor using either
+  of the three options: "Load by editor drag&drop", "Load by drop area
+  drag&drop", or "Load by file picker".
+
+- **Connect to the REPL**: Use any type of serial terminal to connect with the
+  micro:bit, and send a Ctrl+C command to enter the REPL.
+
+- **Script Name**: The name of the script stored on the top right side of the
+  Python Editor. This can be often confused with a search box.
+
+- **Test Case**: A set of test steps to verify a related set of conditions.
+
+
+Preparation
+-----------
+
+Ensure you have everything listed in the "Prerequisites" section.
+
+Each Test Case will focus on a single feature under tests. Make sure to read
+the "Test Cases" opening paragraphs carefully.
+
+
+You will need
+'''''''''''''
+
+- Internet Explorer 10
+
+- A modern version of Edge
+
+- A modern version of Chrome
+
+- A modern version of Firefox
+
+- A text editor (the one included in most operating systems is fine).
+
+- Access to the Python Editor under test
+
+- The test files from
+  https://github.com/bbcmicrobit/PythonEditor/tree/master/tests/manual/test-files
+
+- A micro:bit
+
+- One from the following:
+
+  - A serial terminal program you are comfortable to connect to a micro:bit
+  - The working microREPL installation (https://github.com/ntoll/microrepl)
+  - Mu editor (https://codewith.mu/)
+  - A micro:bit with WebUSB-enabled DAPLink and access to a version of the
+    Python Editor with WebUSB REPL
+
+- If running the tests on Windows 7 you might need the Mbed serial driver
+  installed
+
+
+Tests Execution
+---------------
+
+All Test Cases have to be run in one of the browsers listed in the
+"Preparation" section, and AT LEAST the following Test Cases have to be run
+in all of the other browsers as well:
+
+- Hex file can be generated
+- Generated hex file can be loaded to the editor
+- Python file can be saved
+- Saved Python file can be loaded to the editor
+- Snippets inject code into the Text Editor
+- Help menu works
```

Comment:
clarify 'works' to 'help button expands help menu and links work'

Reply:
"Help menu expands and links work" sounds good, thanks!